app/views/providers/limitations/show.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= page_template(
+      page_title: t('.h1-heading')
+    ) %>
+
+<% if @legal_aid_application.proceeding_types.present? %>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h2 class="govuk-heading-m">
+        <%= t('.h2-heading', count: pluralize(@legal_aid_application.proceeding_types.size, t('.h2-heading'))) %>
```

Comment:
As the translation at `h2-heading` is just "Proceeding" with no sign of a `count` variable, this line should just be:
```erb
<%= t('.h2-heading' %>
```